commands/help.ts

```diff
@@ -13,7 +14,11 @@ export class HelpCommand implements ICommand {
 		private $staticConfig: Config.IStaticConfig) {}
 
 	public enableHooks = false;
-	public allowedParameters: ICommandParameter[] = [new commandParams.StringCommandParameter(this.$injector), new commandParams.StringCommandParameter(this.$injector)];
+	public canExecute(args: string[]): IFuture<boolean> {
+		return Future.fromResult(true);
+	}
+
+	public allowedParameters: ICommandParameter[] = [];
```

Comment:
Imho, we should change our code so that this line should not be needed. But if we agree on this, it is a separate task.